public/index.html

```diff
@@ -47,11 +47,70 @@ <h1 class="app-name">Chat App</h1>
             <md-whiteframe class="md-whiteframe-20dp cont2">
                 <a ng-show="!loggedIn" href="{{loginUri}}" style="text-align: center; top: 50%; left:50%; position: absolute">Log in</a>
 
-                <div ng-include="'start-conv.html'"></div>
 
-                <div ng-include="'conv.html'"></div>
+                <div class="content-container">
+
+                    <md-content class="non-bubble-container" ng-show="loggedIn">
+                        <md-tabs md-dynamic-height md-border-bottom>
+                          <md-tab label="New Conversation">
+                            <md-content class="md-padding flex-col">
+
+                                 <md-content class="conv-users-container">
+                                      <md-list flex>
+                                        
+                                        <md-list-item class="md-1-line" ng-repeat="user in users" ng-click="addUserToConv(user)"  ng-class="isAlreadyInConv(user) ? 'disabled' : 'enabled'">
+                                          <img ng-src="{{user.avatarUrl}}?{{$index}}" class="md-avatar" alt="{{item.who}}" />
+                                          <div class="md-list-item-text" layout="column">
+                                            <p>{{user.name}}</p>
+                                          </div>
+                                        </md-list-item>
+                                      </md-list>
+                                    </md-content>
+
+                                <md-input-container flex="50">
+
+                                  <label>Chat Name</label>
+                                  <input required name="chatName" ng-model="inputConvName">
+                                  <div ng-messages="inputConvName.$error" md-auto-hide="false">
+                                    <div ng-message="required">This is required.</div>
+                                  </div>
+
+                                </md-input-container>
+
+                                <div class="chat-now-btn" ng-click="startConversation(inputConvName)">CHAT NOW</div>
+
+                            </md-content>
+                          </md-tab>
+                          <md-tab ng-if="showConversation" label="{{conv.name}}">
+                            <md-content class="md-padding">
+                              <h1 class="md-display-2">Tab Two</h1>
+                              
+                            </md-content>
+                          </md-tab>
+                        </md-tabs>
+                      </md-content>
+
```

Comment:
A lot of extra whitespace here. Also, did you mean to comment out what is below?